import org.apache.hadoop.fs.PathIOException;
public class FunctionUse4927 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.formatPath(null);
}
}